docs/plugins/advanced_topics/adapted_plugin_guide.md

```diff
@@ -0,0 +1,48 @@
+# Changes to the plugin engine in 0.6.0
+
+In napari 0.6.0 we have made some changes to the default behaviour of legacy "npe1" plugins in napari. This document explains those changes and how they may affect you.
+
+## Context
+
+In mid-2022, we released npe2 - a new napari plugin engine. Mostly invisible to users, npe2 changed the way plugins are built, and how they declare contributions to napari. Because many plugins were already released using the original plugin engine, we continued maintaining all code relating to npe1 plugins, to give developers time to migrate. We also provided [guides and utilities](./npe2_migration_guide.md) for auto-converting npe1 plugins to the new plugin engine.
+
+Through these conversion utilities, the vast majority of npe1 plugins can be automatically loaded by napari in npe2 form, without any changes required from the plugin developer. By enabling the `Preferences -> Plugins -> Use npe2 adaptor` setting, users could also make their own napari installation auto-convert plugins to the new plugin engine, but this setting was off by default.
+
+In the intervening years, adoption of npe2 has improved, and currently, more than 80% of napari plugins are defined using the npe2 engine. As a result, we will start deprecating the npe1 plugin infrastructure, so that we can simplify our codebase and improve maintainability.
+
+## What is changing in 0.6.0?
+
+In napari 0.6.0, the `Use npe2 adaptor` setting will be turned on by default. This means that even if a plugin is still declared using the npe1 engine, napari will auto-convert it to npe2 before loading it.
```

Comment:
```suggestion
In napari 0.6.0: 
- the `Use npe2 adaptor` setting will be turned on by default. 
- if a plugin is still declared using the npe1 engine, napari will auto-convert it to npe2 before loading it.
- users will be notified about any auto-converted plugins when launching napari.
- if there is a conflict with a plugin, the `Use npe2 adaptor` setting may be disabled. This will allow plugins to be loaded using the original npe1 plugin engine.
```